xrmocap/transform/convention/keypoints_convention/__init__.py

```diff
@@ -12,12 +12,14 @@
 from typing import List
 
 from xrmocap.data_structure.keypoints import Keypoints
-from . import campus, human_data, panoptic  # noqa:F401
+from xrmocap.utils.fourdag_utils import all_paf_mapping
```

Comment:
Please move `all_paf_mapping` to `xrmocap/transform/convention/keypoints_convention/paf.py`, and import it by `from .paf import all_paf_mapping`.
There might be circular import if `utils->xrmocap/transform/convention/keypoints_convention->utils`.